tests/test_trainer.py

```diff
@@ -21,7 +21,8 @@ def test_init_no_crash(use_gail, env='CartPole-v1'):
 
 @pytest.mark.parametrize("use_gail", use_gail_vals)
 def test_train_disc_no_crash(use_gail, env='CartPole-v1', n_timesteps=200):
-  trainer = init_trainer(env, use_gail=use_gail)
+  trainer = init_trainer(env, use_gail=use_gail,
+                         rollouts_dir="tests/data/rollouts")
```

Comment:
`tests/data/rollouts` does not exist (on Git).

Reply:
Fixed, but looks like I also need to add `data/rollouts`